.travis.yml

```diff
@@ -1,7 +1,21 @@
 language: python
 python:
   - "2.7"
+  - "3.5"
   - "3.6"
+
+env:
+  - TOXENV=django111
```

Comment:
See my notes on `tox.ini`; we've only been running the non-Django tests in Travis so far, but I think we want to start running all the combinations you have here (at least for 2.7 and 3.5), plus the no-Django test suites that have been running so far.

Also, the `deploy` section is currently mis-configured to try deploying to PyPI multiple times when a release is tagged; please update its `on` conditions to make sure that only one of the test workers tries uploading the new package version.

Reply:
@jmbowman Can you explain the rationale behind needing all the test combinations?